### DISCUSSION TITLE
github classroom 500 error

### DISCUSSION THREAD

**Discussion by putschej** [community/community#130795] [GitHub Education]
### Hi everyone,

I'm unable to accept a github classroom assignment for my class at OSU, and it gives a 500 error every time.

**Comment by RyanHecht** [comment]
Hi @putschej! Do you still receive a 500 error? Due to a recent change, your instructor might need to enable forking private repositories in the Classroom organization -- a message should display to you now if that's the case!

**Reply by putschej** [reply]
Hi, I no longer receive the 500 error.  I'm not sure what my instructor changed, but it is working now.  Thank you for the followup!
